buzzmobile/tools/state_viz/state_viz.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+import cv2
+import os
+import rospy
+import sys
+
+from buzzmobile.msg import CarPose
+from cv_bridge import CvBridge
+from sensor_msgs.msg import Image
+from time import sleep
+
+
+g = {} # globals
+g['start'] = g['auto'] = g['manual'] = g['neutral'] = None
```

Comment:
I generally don't like using this syntax in python, because all the variables point to the same memory. This may be different using dicts.